src/main/java/org/json/simple/parser/Yytoken.java

```diff
@@ -52,6 +52,8 @@ public String toString(){
 		case TYPE_EOF:
 			sb.append("END OF FILE");
 			break;
+		default:

```

Comment:
Great improvement :+1:

Reply:
I think maybe this is not the best option. You should at least log what happened.